tools/layer-zero-example/README.md

```diff
@@ -0,0 +1,175 @@
+
+fill the .env
+
+npx hardhat test --grep "ONFTAdapterTests @hedera @mint" --network hedera_testnet
+npx hardhat test --grep "ONFTAdapterTests @bsc @mint" --network bsc_testnet
+
+npx hardhat test --grep "ONFTAdapterTests @hedera @approve" --network hedera_testnet
+npx hardhat test --grep "ONFTAdapterTests @bsc @approve" --network bsc_testnet
+
+npx hardhat test --grep "ONFTAdapterTests @hedera @send" --network hedera_testnet
+npx hardhat test --grep "ONFTAdapterTests @bsc @send" --network bsc_testnet
+
+wait a couple minutes, the LZ progress can be tracked on https://testnet.layerzeroscan.com/tx/<tx_hash>
+
+npx hardhat test --grep "ONFTAdapterTests @hedera @test" --network hedera_testnet
+npx hardhat test --grep "ONFTAdapterTests @bsc @test" --network bsc_testnet
+
+### HTS Connector
+
+npx hardhat deploy-hts-connector --network hedera_testnet
+npx hardhat deploy-oft --decimals 8 --mint 1000 --network bsc_testnet
+
+npx hardhat set-peer --source <hedera_oft_address> --target <bsc_oft_address> --network hedera_testnet
+npx hardhat set-peer --source <bsc_oft_address> --target <hedera_oft_address> --network bsc_testnet
+
+fill the .env
+
+npx hardhat test --grep "HTSConnectorTests @hedera @approve" --network hedera_testnet
+
+npx hardhat test --grep "HTSConnectorTests @hedera @send" --network hedera_testnet
+npx hardhat test --grep "HTSConnectorTests @bsc @send" --network bsc_testnet
+
+wait a couple minutes, the LZ progress can be tracked on https://testnet.layerzeroscan.com/tx/<tx_hash>
+
+npx hardhat test --grep "HTSConnectorTests @hedera @test" --network hedera_testnet
+npx hardhat test --grep "HTSConnectorTests @bsc @test" --network bsc_testnet
+
+### HTS Adapter
+
+npx hardhat create-hts-token --network hedera_testnet
+npx hardhat deploy-erc20 --decimals 8 --mint 1000 --network bsc_testnet
+
+npx hardhat deploy-oft-adapter --token <erc20_hedera_address> --network hedera_testnet
+npx hardhat deploy-oft-adapter --token <erc20_bsc_address> --network bsc_testnet
+
+npx hardhat set-peer --source <hedera_oft_adapter_address> --target <bsc_oft_adapter_address> --network hedera_testnet
+npx hardhat set-peer --source <bsc_oft_adapter_address> --target <hedera_oft_adapter_address> --network bsc_testnet
+
+fill the .env
+
+npx hardhat test --grep "HTSAdapterTests @hedera @fund-and-approve" --network hedera_testnet
+npx hardhat test --grep "HTSAdapterTests @bsc @fund-and-approve" --network bsc_testnet
+
+npx hardhat test --grep "HTSAdapterTests @hedera @send" --network hedera_testnet
+npx hardhat test --grep "HTSAdapterTests @bsc @send" --network bsc_testnet
+
+wait a couple minutes, the LZ progress can be tracked on https://testnet.layerzeroscan.com/tx/<tx_hash>
+
+npx hardhat test --grep "HTSAdapterTests @hedera @test" --network hedera_testnet
+npx hardhat test --grep "HTSAdapterTests @bsc @test" --network bsc_testnet
+
+### WHBAR flow
```

Comment:
This shouldn't be in the layer zero dir in it's final form.
Move this to `tools/whbar`

Reply:
Agreed. I put it there because the WHBAR example is on another branch and if I create a duplication of it in that PR, there will be future conflicts to resolve. Will move it as you suggested after I finalize the README.